packages/tigervnc/vncserver

```diff
@@ -60,7 +60,7 @@ $xauthorityFile = "$ENV{XAUTHORITY}" || "$ENV{HOME}/.Xauthority";
 
 $xstartupFile = $vncUserDir . "/xstartup";
 $defaultXStartup
-    = ("#!/data/data/com.termux/files/usr/bin/sh\n".
+    = ("#!/usr/bin/sh\n".
```

Comment:
Does this work?

Reply:
By default, libtermux-exec.so is enabled since login, so the `/usr` shebang is handled by libtermux-exec.so

However, Still i need to mention @Grimler91 about this as i, myself is not sure yet whenever this shebang is allowed or no.

Reply:
Oh, I didn't notice that because I am using a custom environment. That said, I would prefer `sed`ing this also.